website/docs/diff.md

```diff
@@ -9,15 +9,16 @@ sidebar_position: 6
 1. FerretDB uses the same protocol error names and codes, but the exact error messages may be different in some cases.
 2. FerretDB does not support NUL (`\0`) characters in strings.
 3. Document keys must not contain `$` sign.
-4. Database and collection names restrictions:
+4. Document keys must not contain `.` sign.
```

Comment:
What I propose to do is to have one file in dance per numbered item. In that case, it would be easier to combine those two points

Reply:
Fixed.